src/workerd/api/worker-rpc.c++

```diff
@@ -173,14 +173,20 @@ public:
         (Worker::Lock& lock) mutable -> kj::Promise<void> {
 
       jsg::Lock& js = lock;
-      // JS RPC is not enabled on the server side, we cannot call any methods.
-      JSG_REQUIRE(FeatureFlags::get(js).getJsRpc(), TypeError,
-          "The receiving Worker does not allow its methods to be called over RPC.");
 
       auto handler = KJ_REQUIRE_NONNULL(lock.getExportedHandler(entrypointName, ctx.getActor()),
                                         "Failed to get handler to worker.");
       auto handle = handler->self.getHandle(lock);
 
+      if (handler->missingSuperclass) {
+        // JS RPC is not enabled on the server side, we cannot call any methods.
+        JSG_REQUIRE(FeatureFlags::get(js).getJsRpc(), TypeError,
+            "The reciving Durable Object does not support RPC, because its class was not declared "
```

Comment:
`reciving` -> `receiving` since this is a user facing error.

Reply:
fixup: 496e33c6